internal/resources/sourcesecret/data_sourcesecret_test.go

```diff
@@ -41,39 +41,6 @@ func TestAcceptanceForSourceSecretDataSource(t *testing.T) {
 				Config: testConfig.getTestSourceSecretDataSourceBasicConfigValue(commonscope.ClusterScope, spec.SSHKey),
 				Check:  testConfig.checkSourceSecretDataSourceAttributes(),
 			},
-			{
```

Comment:
Don't we need this test? @Shubbhang351

Reply:
second tests I've added last time just to test spec SSH Type, which can be checked through update call so I've merged the tests for both spec type.

Reply:
Sure.